src/test/resources/org/apache/ibatis/migration/TemplateReaderTest.sql

```diff
@@ -0,0 +1,19 @@
+--
+--    Copyright 2010-2018 the original author or authors.
```

Comment:
We've given up on getting this merged, right?
```
          <groupId>com.mycila</groupId>
          <artifactId>license-maven-plugin</artifactId>
```

It's defined in the parent pom.  Disable it.